loader/loader-for-all-content/loader-for-all-content/Pages/Index.razor

```diff
@@ -0,0 +1,34 @@
+﻿@page "/"
+
+<h1>This page is OK - you can't scroll and interact with the content behind the loader container</h1>
+
+<style>
+    @* set all top-level elements in the layout to height 100% so they match the viewport *@
+    html, body, .page, .main{
+        height: 100%;
+        min-height: 100%;
+        max-height: 100%;
```

Comment:
do we need this?

Reply:
Sometimes, they are a safety guard for layouts that are not implemented too well and can have insufficient size or leak content otherwise. Will add comments to denote this.